opentech/apply/determinations/templates/determinations/determination_form.html

```diff
@@ -33,8 +33,8 @@ <h2>{{ value }}</h2>
         <input class="button button--secondary button--white" type="submit" value="Save Draft" name="{{ form.draft_button_name }}" />
     </form>
     {% for type, message in canned_messages.items %}
-        <div id="determination-message-{{ type }}">
-        {{ message|bleach }}
+        <div class="is-hidden" data-type="{{ type }}" id="determination-message-{{ type }}">
```

Comment:
I'm not using `id="determination-message-{{ type }}"` so feel free to remove if not required